src/kaggle/api/kaggle_api_extended.py

```diff
@@ -1929,8 +1929,11 @@ def download_file(self,
         size = int(response.headers['Content-Length'])
         size_read = 0
         open_mode = 'wb'
-        remote_date = datetime.strptime(response.headers['Last-Modified'],
-                                        '%a, %d %b %Y %H:%M:%S %Z')
+        try:

```

Comment:
instead of a try/except, consider using:

```
if 'Last-Modified' in response.headers:
 remote_date = datetime.strptime(response.headers['Last-Modified'],
                                            '%a, %d %b %Y %H:%M:%S %Z')
else:
  remote_date = datetime.now()
```

Reply:
Or actually, you can follow this pattern instead: https://github.com/Kaggle/kaggle-api/blob/dce6447119de56820eb8ce50b2a873ec46c3ba27/src/kaggle/api/kaggle_api_extended.py#L4124-L4127